io/io/test/TROSharedMemFileTests.cxx

```diff
@@ -0,0 +1,37 @@
+#include "TROSharedMemFile.h"
+
+#include "TError.h"
+#include "TMemFile.h"
+#include "TTree.h"
+
+#include "gtest/gtest.h"
+
+TEST(TFileMerger, CreateWithTFilePointer)
```

Comment:
not sure how, but it would be nice to test that two `TROSharedMemFiles` actually read from a shared memory buffer